php/routes.php

```diff
@@ -26,4 +26,42 @@
             ]
         );
     });
+
+    CBPRouter::code('/code', function ($payload) {
```

Comment:
It should be a GET request to the server since we are just getting data, so it shoudl just be `CBPRouter::get(`